content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md

```diff
@@ -124,31 +124,31 @@
 
 - **Accountable**: PM
 - **Responsible**: EM / Team
-- **Expectations**: PM is accountable for making sure the teams iterations are aligned with high level goals
+- **Expectations**: PM is accountable for making sure the teams iterations are aligned with high-level goals
 
 #### Providing guidance on prioritization
 
 - **Accountable**: PM
 - **Responsible**: Team
-- **Expectations**: PM is accountable for making sure we align with high level company goals, EM is expected to work closely with PM to ensure we make progress towards it and inform on technical details. The team should be proposing issues and priorities for an iteration.
+- **Expectations**: PM is accountable for ensuring we align with high-level company goals. EM is expected to work closely with PM to ensure we progress towards it and inform on technical details. The team should be proposing issues and priorities for an iteration.
 
 #### Capacity planning
 
 - **Accountable**: EM
 - **Responsible**: Team
-- **Expectations**: EMs are accountable from making sure we balance the roadmap items and passive items (backlog/tech debt) with engineering capacity
+- **Expectations**: EMs are accountable for making sure we balance the roadmap items and passive items (backlog/tech debt) with engineering capacity
 
 #### Ticket refinement (scoping, measures of success)
 
 - **Accountable**: EM
 - **Responsible**: PM
-- **Expectations**: EMs are accountable for making sure the GitHub issues have all the required information for engineers to pick up and get started.
+- **Expectations**: EMs are accountable for ensuring the GitHub issues have all the required information for engineers to pick up and get started.
```

Comment:
Similarly should be "The EM..." 
Also should be "ensuring that GitHub issues..."